Polygon_mesh_processing/examples/Polygon_mesh_processing/isotropic_remeshing_example.cpp

```diff
@@ -62,5 +73,13 @@ int main(int argc, char* argv[])
 
   std::cout << "Remeshing done." << std::endl;
 
+  std::cout << "took " << t.time() << " sec" << std::endl;
+  t.reset();
+
+  std::ofstream out("out.off");
+  out.precision(17);
+  //out << mesh << std::endl;
+  out.close();
+
```

Comment:
@MaelRL just spotted this.   @janetournois shall we write a mesh or shall we just go back to the original file and also kick out the timer.

Reply:
Since it is the "main" example for isotropic remeshing, visible in the user manual, I would remove this and the timer too.

Reply:
If you look at the other examples it seems that those who somehow modify the mesh or produce a mesh (union), at the end write the mesh to a file.   We might unify.   I also do not like the prefix or suffix `example`   in the file names.